_conferences/devfest-czech-republic-2018.md

```diff
@@ -0,0 +1,12 @@
+---
+name: "DevFest CZ 2018"
```

Comment:
The name should be "GDG DevFest" to match the others. The country and year are already displayed.

Reply:
Edited in latest update as GDG DevFest